src/modules/AutoinstGeneral.rb

```diff
@@ -180,16 +186,14 @@ def Import(settings)
       settings = deep_copy(settings)
       SetModified()
       Builtins.y2milestone("General import: %1", settings)
-      @mode = Ops.get_map(settings, "mode", {})
-      @cio_ignore = Ops.get_boolean(settings, "cio_ignore", true)
-      @signature_handling = Ops.get_map(settings, "signature-handling", {})
-      @askList = Ops.get_list(settings, "ask-list", [])
-      @proposals = Ops.get_list(settings, "proposals", [])
-      @storage = Ops.get_map(settings, "storage", {})
+      @mode = settings.fetch("mode", {})
```

Comment:
Well, `foo.fetch(bar, whatever)` is closer to `Ops.get_xx(foo, bar, whatever)` than `foo[bar] || whatever`. The latter would act different if the key is there but the value is `nil` or `false`. So I think it's wise to use `#fetch` when replacing this kind of legacy code.